### DISCUSSION TITLE
What do you mean by “Better support for mobile screens” ?

### DISCUSSION THREAD

**Discussion by amrpyt** [qunash/chatgpt-advanced#127] [Q&A] [answered]
**What do you mean by “Better support for mobile screens” ?**

Can i run the extension on my phone? (Ios)

**Comment by qunash** [comment, marked as answer]
Hi, the toolbar is now scrollable when it doesn't fit on the screen. The extension used to work in Kiwi browser on mobile, but recently stopped. Haven't had a chance to look into it yet. There may be are some other mobile browser that support Google chrome extensions.
